services/gateway/tests/constants/registerApi.js

```diff
@@ -429,11 +429,11 @@ const expectedResponseForTransformRequest = {
 	chainID: undefined,
 	chainName: undefined,
 	limit: 10,
-	network: 'devnet,alphanet',
+	network: 'devnet,devnet',
 	offset: 0,
 	search: undefined,
 	sort: 'chainName:asc',
-	tokenID: '0300000000000000,0400000000000000',
+	tokenID: '0400000000000000,0400000000000000',
```

Comment:
would it be better to use betanet (02000000) instead of mentioning devnet twice?